static/src/javascripts/projects/membership/payment-form.js

```diff
@@ -50,44 +44,53 @@ define([
             TIER_FIELD: 'js-tier-field',
             UPDATING: 'js-updating'
         },
-        DOM: {},
         data: {
             CARD_TYPE: 'data-card-type'
         }
     };
 
+    function StripePaymentForm(form, successElement, url) {
```

Comment:
Please document that `successElement` is expected to be a bonzo wrapped set! Or maybe better, expect it to be an actual DOM element and wrap it inside the class